src/main/java/uk/ac/cam/cl/dtg/segue/dos/users/UserContext.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.cam.cl.dtg.segue.dos.users;
+
+public class UserContext {
+    public enum STAGE {gcse, a_level, further_a, university, none}
+    public enum EXAM_BOARD {AQA, OCR, CIE, EDEXCEL, EDUCAS, WJEC, OTHER, NONE}
```

Comment:
We might not need the "None" options here, it could be recorded as null but still update the date. I will ask the content team on Thursday whether or not they need a None option for Exam Board in particular.

Reply:
Why are these enum class names not `CamelCase` like Java classes are meant to be? I am sure it is for a reason, I'm just wondering!

Reply:
Actually looks like CamelCase is the convention for enums, I'll change to that.
I tend to think of enums as namespaces for constants which leads me to think that they should also be SCREAMING_SNAKE_CASE like constants.